README.md

```diff
@@ -1,4 +1,4 @@
-# Image [![Build Status](https://travis-ci.org/image-rs/image.svg?branch=master)](https://travis-ci.org/image-rs/image) [![Gitter](https://badges.gitter.im/image-rs/image.svg)](https://gitter.im/image-rs/image?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+# Image [![crates.io](https://img.shields.io/crates/v/image.svg)](https://crates.io/crates/image) [![Build Status](https://travis-ci.org/image-rs/image.svg?branch=master)](https://travis-ci.org/image-rs/image) [![Gitter](https://badges.gitter.im/image-rs/image.svg)](https://gitter.im/image-rs/image?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
 Maintainers: @nwin, @ccgn
```

Comment:
I guess this is kinda out of scope for this PR, but we should probably change these as both nwin an ccgn have been inactive for some time.

Reply:
Agree. Maybe just drop the Maintainers line, or add a link to https://github.com/orgs/image-rs/people or something?

Reply:
Link to the team sounds like a possible solution?

Reply:
I opened #950 for the maintainer part. Could we merge this PR ?

Reply:
Sure